### PR TITLE
Replace deprecated 'inputs' property of IENetwork and ExecutableNetwork in Python demos

### PR DESCRIPTION
**Pull request by AnthonyQuantum**
openvinotoolkit/openvino#637

### CODE REVIEW

**Comment by AnthonyQuantum** [comment]
@IRDonch,
the OpenVino verison in Jenkins does not have features from openvinotoolkit/openvino#637 (merged 7 days ago).
Is it possible to update the OpenVino there?

File: demos/python_demos/action_recognition/action_recognition_demo/models.py [hunk 1]
```diff
@@ -85,14 +85,14 @@ def __init__(self, model_xml, model_bin, ie_core, target_device, num_requests, b
         print("Reading IR...")
         self.net = ie_core.read_network(model_xml, model_bin)
         self.net.batch_size = batch_size
-        assert len(self.net.inputs.keys()) == 1, "One input is expected"
+        assert len(self.net.input_info.keys()) == 1, "One input is expected"
```
Comment (IRDonch) [inline demos/python_demos/action_recognition/action_recognition_demo/models.py:88]: While you're at it, could you also replace `len(self.net.input_info.keys())` with `len(self.net.input_info)`?

**Comment by IRDonch** [comment]
> Is it possible to update the OpenVino there?

Yeah, I'll try to do that soon.

File: demos/python_demos/action_recognition/action_recognition_demo/models.py (hunk 1 above)
Comment (AnthonyQuantum) [reply demos/python_demos/action_recognition/action_recognition_demo/models.py]: Done

**Comment by AnthonyQuantum** [comment]
Jenkins please retry a build

**Comment by IRDonch** [comment]
FYI, I haven't updated CI configuration yet. I'll restart the build myself when I do so.

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Comment by IRDonch** [comment]
It looks like `multi_camera_multi_person_tracking` freezes with your changes. Could you investigate what's going on?

**Comment by IRDonch** [comment]
There have been a few new demos merged since you started this, which also use the deprecated property. Could you update them, as well?

Also, it looks like `.output` is similarly deprecated. Could you replace it too?

**Comment by AnthonyQuantum** [comment]
@IRDonch 

> There have been a few new demos merged since you started this, which also use the deprecated property. Could you update them, as well?
> 
> Also, it looks like `.output` is similarly deprecated. Could you replace it too?

opencv/open_model_zoo#1225

**Pull request merged**